compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -355,12 +355,18 @@ class Definitions {
     lazy val Predef_undefinedR: TermRef = ScalaPredefModule.requiredMethodRef(nme.???)
     def Predef_undefined(implicit ctx: Context): Symbol = Predef_undefinedR.symbol
 
-  def SubTypeClass(implicit ctx: Context): Symbol =
+  def SubTypeClass(implicit ctx: Context): ClassSymbol =
     if (isNewCollections)
       ctx.requiredClass("scala.<:<")
     else
       ScalaPredefModule.requiredClass("<:<")
 
+  def DummyImplicitClass(implicit ctx: Context): ClassSymbol =
```

Comment:
For my own understanding: Why was this class moved?

Reply:
I moved it along with <:< and =:= in https://github.com/scala/scala/pull/7350 to keep Predef as lean as possible, classes can be defined at the top-level so I didn't see why these one in particulars should be nested in Predef.